bpf/headers/bpf_helpers.h

```diff
@@ -10,7 +10,7 @@
  */
 #include "bpf_helper_defs.h"
 
-#define __uint(name, val) int(*name)[val]
+#define __uint(name, val) int (*name)[val]
```

Comment:
I think you can just undo this one and clang-format will pass (it should also build fine).
```suggestion
#define __uint(name, val) int(*name)[val]
```